src/main/frontend/ui.cljs

```diff
@@ -214,7 +214,7 @@
            [:div.flex-shrink-0
             svg]
            [:div.ml-3.w-0.flex-1
-            [:div.text-sm.leading-5.font-medium {:style {:margin 0}
+            [:div.text-sm.leading-5.font-medium {:style {:margin 0 :white-space "pre-line"}
```

Comment:
tailwindcss has a `whitespace-pre-line`
ref:  https://tailwindcss.com/docs/whitespace

Reply:
Thanks for the reminder, I've modified the code.